pkg/apis/work/v1alpha1/binding_types.go

```diff
@@ -6,6 +6,26 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
+const (
+	// ResourceKindResourceBinding is kind name of ResourceBinding
+	ResourceKindResourceBinding = "ResourceBinding"
+	// ResourceSingularResourceBinding is singular name of ResourceBinding
+	ResourceSingularResourceBinding = "resourcebinding"
+	// ResourcePluralResourceBinding is plural name of ResourceBinding
+	ResourcePluralResourceBinding = "resourcebindings"
+	// ResourceNamespaceScopedResourceBinding indicates if ResourceBinding is NamespaceScoped
+	ResourceNamespaceScopedResourceBinding = true
+
+	// ResourceKindClusterResourceBinding is kind name of ClusterResourceBinding
+	ResourceKindClusterResourceBinding = "ClusterResourceBinding"
+	// ResourceSingularClusterResourceBinding is singular name of ClusterResourceBinding
+	ResourceSingularClusterResourceBinding = "clusterresourcebinding"
+	// ResourcePluralClusterResourceBinding is kind plural of ClusterResourceBinding
+	ResourcePluralClusterResourceBinding = "clusterresourcebindings"
+	// ResourceNamespaceScopedClusterResourceBinding indicates if ClusterResourceBinding is NamespaceScoped
+	ResourceNamespaceScopedClusterResourceBinding = false
+)
+
```

Comment:
The `v1alpha1` version has been deprecated, do we still have to generate the API for it?